README.md

```diff
@@ -22,9 +22,7 @@ in the menu component's stories.
 
 ## Demo App (Deprecated)
 
-Run `npm run demo` for a legacy demo server. Use this as reference when
-converting demo pages to stories. Do not add new demo pages. New compoents or
-examples of existing components should be done as Storybook stories.
+Run `npm run demo` for a legacy demo server. Converted to storybook.
```

Comment:
I removed all the old demo code. It's been fully replaced by storybook for some time.